### DISCUSSION TITLE
What is the default username/password after installation?

### DISCUSSION THREAD

**Discussion by nacesprin** [thedevs-network/kutt#601] [General]
.

**Comment by poeti8** [comment]
You need to sign up and update the `.env` with your email address if you want your user to be admin.

**Reply by voron69-bit** [reply]
> You need to sign up and update the `.env` with your email address if you want your user to be admin.

Is this just the ADMIN_EMAILS env variable? Because I've done this. Signed up. Logged out. Changed the ENV variable and restarted KUTT. Logged back in, there is no admin panel etc. Did I do something wrong here?

**Reply by poeti8** [reply]
The admin panel is on version 3. On version 3 there's no ADMIN_EMAILS, you create the admin account when you start the app for the first time.

If you're not starting the app for the first time, them manually find the user on your database and set the value for the role column to `ADMIN` (all uppercase).
